src/base_client.ts

```diff
@@ -6,12 +6,22 @@ export class LuneClient {
     protected client: AxiosInstance
     protected config: ClientConfig
 
-    constructor(apiKey: string, baseUrl: string = 'https://api.lune.co', apiVersion: string = '1') {
+    constructor(
+        apiKey: string,
+        baseUrl: string = 'https://api.lune.co',
+        apiVersion: string = '1',
+        targetAccount?: string,
```

Comment:
I'd keep the terminology simple: `account`.
`account` is unambiguous.
`target` can be confusing.

Reply:
Yes, I agree, just an `account` will be better I think